packages/Search/src/DTK_LinearBVH_decl.hpp

```diff
@@ -33,7 +33,7 @@ struct TreeTraversal;
  * Bounding Volume Hierarchy.
  */
 template <typename DeviceType>
-struct BVH
+class BVH
```

Comment:
OK, so in order to be consistent with the explicit instantiation in the _def file ([here](https://github.com/Rombur/DataTransferKit/blob/d77f255a1be53d29c919dddd48865484c94841ad/packages/Search/src/DTK_LinearBVH_def.hpp#L98)).